extensions/amp-story/1.0/amp-story.js

```diff
@@ -2694,11 +2697,29 @@ export class AmpStory extends AMP.BaseElement {
       win.CSS && win.CSS.supports && win.CSS.supports('display', 'grid')
     );
   }
+
+  /**
+   * Downloads Material Icons if bling links are present in the story
+   */
+  getBlingLinkIcons_() {
+    const blingLinks = this.win.document.querySelector('amp-story-bling-link');
+    if (!blingLinks) {
+      return;
+    }
+    const fontsEl = this.win.document.createElement('link');
+    fontsEl.setAttribute(
+      'href',
+      'https://fonts.googleapis.com/icon?family=Material+Icons'
+    );
+    fontsEl.setAttribute('rel', 'stylesheet');
+    this.win.document.head.appendChild(fontsEl);
+  }
 }
 
 AMP.extension('amp-story', '1.0', AMP => {
   AMP.registerElement('amp-story', AmpStory, CSS);
   AMP.registerElement('amp-story-access', AmpStoryAccess);
+  AMP.registerElement('amp-story-bling-link', AmpStoryBlingLink);
```

Comment:
OK for now, but I suspect we don't actually want to call it this 😄

Reply:
Open to ideas haha.

Reply:
Not saying I actually have any 😅 

I'll sync with some folks on the monetization side who may be a bit closer to what publishers would actually expect.  Definitely not blocking for now